transaction/README.md

```diff
@@ -0,0 +1,9 @@
+## Transaction
+
+Everything that has to do with sending transactions successfully. It is basically a smart version of `/transfer` (soon deprecated). It is composed of 2 main objects:
+
+- Courier: which is in charge of signing and sending the transactions to the blockchain with the given parameters.
+- Depot: It is the main object and it is very smart. Its job is to send the transactions using the courier, keep track of them (storing the progess safely on a storage), and making sure that the transactions gets mined.  
```

Comment:
Let's word this a bit better. Let's drop the being smart part and lets words this maybe something like this:

Depot: It works in a similar fashion to a regular delivery depot in real life. It has `Workers` which check for incoming transactions, pack them and give them away to `Couriers` which try to deliver the transaction. Transaction state is tracked just like a regular package would be, trying to deliver it if it was not accepted etc.